.vscode/settings.json

```diff
@@ -39,5 +39,10 @@
     "python.testing.cwd": "${workspaceFolder}/tests",
     "cSpell.words": [
         "crowdsourcing"
-    ]
+    ],
+    "isort.args":["--profile", "black"],
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnSave": true
+    }
```

Comment:
Formatting is only enabled for some new python components, listed [here](https://github.com/cvat-ai/cvat/blob/develop/dev/format_python_code.sh).